libethcore/Transaction.cpp

```diff
@@ -130,6 +133,14 @@ Address const& TransactionBase::sender() const
 	return m_sender;
 }
 
+SignatureStruct const& TransactionBase::signature() const
+{ 
+	if (!m_vrs)
```

Comment:
yeah by null I guess I meant  an uninittialized struct.
Having this function to throw is not something that I would have expected if i would use it, 
but if the others are fine by throwing here, that's ok